index.d.ts

```diff
@@ -4,7 +4,7 @@ declare module "focus-trap-react" {
 
   // Extend AllHTMLAttributes to provide completions on DOM properties wherever
   // React does.
-  export interface Props extends React.AllHTMLAttributes<any> {
+  interface Props extends React.AllHTMLAttributes<any> {
```

Comment:
@dsifford somehow I didn't notice last time that you removed the export declaration here. Is it possible to restore that at some point? When declaring an HOC or the like, it's useful to be able to name the props interface.

Reply:
Ah, yep good point... It'll require another rework though. Can't be done in the form that exists currently. 

I'll submit another PR for it in a couple mins.